app/app_xdg.go

```diff
@@ -32,6 +37,107 @@ func (a *fyneApp) OpenURL(url *url.URL) error {
 	return cmd.Start()
 }
 
+func findCurrentWM() string {
+
+	wm := os.Getenv("XDG_CURRENT_DESKTOP")
+	if wm == "" {
+		wm = os.Getenv("DESKTOP_SESSION")
+	}
+	wm = strings.ToLower(wm)
+	return wm
+}
+
+func findThemeVariant() fyne.ThemeVariant {
+	wm := findCurrentWM()
+	switch wm {
+	case "gnome", "xfce", "unity", "gnome-shell", "gnome-classic", "mate", "gnome-mate":
+		return findGnomeThemeVariant()
+	case "kde", "kde-plasma", "plasma":
+		return findKdeThemeVariant()
+	default:
+		return theme.VariantDark
+	}
+}
+
+// find the current KDE theme variant. At this time, no solution.
+func findKdeThemeVariant() fyne.ThemeVariant {
```

Comment:
I think the name should be `findKDEThemeVariant` with casing?

Reply:
fixed